src/hotspot/share/utilities/globalDefinitions_visCPP.hpp

```diff
@@ -102,7 +102,7 @@ inline int g_isfinite(jfloat  f)                 { return _finite(f); }
 inline int g_isfinite(jdouble f)                 { return _finite(f); }
 
 // Formatting.
-#define FORMAT64_MODIFIER "I64"
+#define FORMAT64_MODIFIER "ll"
```

Comment:
Interesting - this seems un-needed and should be replaced in its one use by `INT64_FORMAT_X_0` - but separate PR I think.